src/Symfony/Component/DependencyInjection/Attribute/Closure.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Attribute;
+
+#[\Attribute(\Attribute::TARGET_FUNCTION)]
+class Closure
```

Comment:
I think we should remove the class - this should remain internal and not having a class might be the best way to keep it internal

Reply:
OK, let's keep this attribute internal.